internal/cmd/workspaces_test.go

```diff
@@ -29,6 +29,34 @@ func Test_workspaces_ls(t *testing.T) {
 	res.stdoutUnmarshals(t, &workspaces)
 }
 
+func Test_workspaces_ls_all(t *testing.T) {
+	skipIfNoAuth(t)
+	for _, test := range []struct {
+		name    string
+		command []string
+		assert  func(r result)
+	}{
+		{
+			name:    "simple list",
+			command: []string{"workspaces", "ls", "--all"},
+			assert:  func(r result) { r.success(t) },
```

Comment:
Sanity check: This isn't actually checking the output - just that the command exited with 0?

Reply:
Essentially, yeah. This mini test framework "runs" the commands in-process and checks if the command fn returned an error.